CHANGELOG.md

```diff
@@ -31,9 +31,10 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#5409](https://github.com/thanos-io/thanos/pull/5409) S3: Add option to force DNS style lookup.
 - [#5352](https://github.com/thanos-io/thanos/pull/5352) Cache: Add cache metrics to groupcache.
 - [#5391](https://github.com/thanos-io/thanos/pull/5391) Receive: Add relabeling support.
-- [#5408](https://github.com/thanos-io/thanos/pull/5391) Receive: Add support for consistent hashrings.
+- [#5408](https://github.com/thanos-io/thanos/pull/5408) Receive: Add support for consistent hashrings.
 - [#5391](https://github.com/thanos-io/thanos/pull/5391) Receive: Implement api/v1/status/tsdb.
 - [#5424](https://github.com/thanos-io/thanos/pull/5424) Receive: export metrics regarding size of remote write requests
+- [#5440](https://github.com/thanos-io/thanos/pull/5440) HTTP metrics: export number of in-flight HTTP requests
```

Comment:
super nit but I think the pattern seems to be to end the sentence with `.` as in the other changelog entries?

Reply:
Fixed this and other changelog entries lacking the period in [7db4419](https://github.com/thanos-io/thanos/pull/5440/commits/7db44191909a20624bf98acf1f2f679455944ac7).